include/aktualizr-lite/api.h

```diff
@@ -371,6 +371,10 @@ class AkliteClient {
 
  private:
   void Init(Config &config, bool finalize = true, bool apply_lock = true);
+  CheckInResult checkInFailure(CheckInResult::Status check_status, std::string err_msg) const;
```

Comment:
I would try and make these utility functions as a static non-class methods, if possible, and defined in api.cc, so the header is not changed and the api is not polluted with the helper functions.

Reply:
Changed all new private methods to static functions. `api.h` has no changes in the current PR version.